runtime/onert/api/include/nnfw.h

```diff
@@ -246,8 +246,11 @@ NNFW_STATUS nnfw_set_input(nnfw_session *session, uint32_t index, NNFW_TYPE type
 /**
  * @brief       Set output buffer
  *
- * This function should be called after {@link nnfw_prepare}, and before first inference
- * on session by {@link nnfw_run}. Application can reuse buffer for many inferences.
+ * This function must be called after {@link nnfw_prepare}, \p buffer given to this function can be
+ * reused for many inferences. \p length must be greater or equal than the operand requires. An
+ * output operand can have unspecified shape and deduced dynamically during the execution. You must
+ * provide \p buffer large enough. To specify an optional input, you can either not call this
+ * for that input or call this with \p buffer of NULL and \p length of 0.
```

Comment:
`To specify an optional input, you can either not call this
 * for that input or call this with \p buffer of NULL and \p length of 0.`
Maybe not relevant statement here?